ng-dev/utils/testing/virtual-git-client.ts

```diff
@@ -99,8 +99,14 @@ export class VirtualGitClient extends AuthenticatedGitClient {
 
   /** Handler for the `git push` command. */
   private _push(args: string[]) {
-    const [repoUrl, refspec] = parseArgs(args, {boolean: ['q']})._;
-    const ref = this._unwrapRefspec(refspec);
+    const [repoUrl, refspec] = yargs(args)
```

Comment:
Is there a way to manage `repoUrl` and `refspec` as "positional" here? that would save us from adding the type casts.

Reply:
You'd need to define commands. 
Example: https://github.com/yargs/yargs/blob/master/docs/advanced.md#advanced-topics

Reply:
which I guess could make sense as it's basically mocking a command, but I'm fine either way.

Reply:
I can get back to replacing this next week.